Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UITransform/UITransformUpdateSystem.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.Throttling;
+using DCL.Diagnostics;
+using DCL.ECSComponents;
+using DCL.SDKComponents.SceneUI.Components;
+using DCL.SDKComponents.SceneUI.Utils;
+using ECS.Abstract;
+using ECS.Groups;
+using SceneRunner.Scene;
+using UnityEngine.UIElements;
+
+namespace DCL.SDKComponents.SceneUI.Systems.UITransform
+{
+    [UpdateInGroup(typeof(SyncedSimulationSystemGroup))]
+    [UpdateAfter(typeof(UITransformSortingSystem))]
+    [LogCategory(ReportCategory.SCENE_UI)]
+    [ThrottlingEnabled]
+    public partial class UITransformUpdateSystem : BaseUnityLoopSystem
+    {
+        private readonly UIDocument canvas;
+        private readonly ISceneStateProvider sceneStateProvider;
+
+        public UITransformUpdateSystem(World world, UIDocument canvas, ISceneStateProvider sceneStateProvider) : base(world)
+        {
+            this.canvas = canvas;
+            this.sceneStateProvider = sceneStateProvider;
+        }
+
+        protected override void Update(float _)
+        {
+            UpdateUITransformQuery(World);
+            CheckUITransformOutOfSceneQuery(World);
+        }
+
+        [Query]
+        [All(typeof(PBUiTransform), typeof(UITransformComponent))]
```

Comment:
`All` is redundant as you included all the same types as arguments

Reply:
@mikhail-dcl what do you mean? Can I remove the `[All(typeof(PBUiTransform), typeof(UITransformComponent))]` line if I have the same 2 arguments in the function signature?